package.json

```diff
@@ -40,7 +40,7 @@
   },
   "devDependencies": {
     "chai": "^4",
-    "eslint": "^7",
+    "eslint": "^8",
```

Comment:
This explains why `eslint` wasn't getting upgraded by Dependabot -- it was that peer dependency in our config that was holding it back!